Sources/SwiftParser/Statements.swift

```diff
@@ -255,7 +254,14 @@ extension Parser {
       let pattern = self.parseMatchingPattern(context: .matching)
       kind = .pattern(caseKeyword, pattern)
     } else {
-      let letOrVar = self.consumeAnyToken()
+      let letOrVar: RawTokenSyntax
+
+      if self.at(.identifier) {
+        letOrVar = RawTokenSyntax(missing: lastBindingKind!.tokenKind, text: lastBindingKind!.tokenText, arena: self.arena)
+      } else {
+        letOrVar = self.consumeAnyToken()
```

Comment:
I know this has been `consumeAnyToken` before but if you’ve got time could you try if you could change this to `self.expect(.let, .var)`. That just makes it a lot clearer what we expect `letOrVar` to be. `consumeAnyToken` is pretty much an anti-pattern in 90% of all cases that I would like to get rid of in general.